specs/subresourceintegrity/spec.markdown

```diff
@@ -332,8 +332,8 @@ One should note that being a [secure document][] (e.g. a document delivered
 over HTTPS) is not necessary for the use of integrity validation. Because
 resource integrity is only an application level security tool, and it does not
 change the security state of the user agent, a [secure document] is
-unnecessary. However, if integrity is used in other than a [secure document][] (e.g.
-a document delivered over HTTP), authors should be aware that the integrity
+unnecessary. However, if integrity is used in somethig else than a [secure document][]
+(e.g. a document delivered over HTTP), authors should be aware that the integrity
```

Comment:
I learned to put a comma after both, but I'm not a native speaker \*shrugs\*.
Reading through https://english.stackexchange.com/questions/16172/should-i-always-use-a-comma-after-e-g-or-i-e I'd still go for a comma everywhere :-)